app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -62,12 +72,43 @@ protected function indexData($request)
         ];
     }
 
+    protected function transformIndexItems($items)
+    {
+        // If we're in the browser, don't transform the items
+        if (property_exists($items, 'path')) {
+            return $items;
+        }
+        return $items->toTree();
+    }
+
+    protected function indexItemData($item)
+    {
+        return $item->children ? [
+            'children' => $this->getIndexTableData($item->children),
+        ] : [];
+    }
+
+    private function getParents($exceptPage = null)
+    {
+        return $this->repository->whereNotIn('id', is_null($exceptPage) ? [] : [$exceptPage])->withDepth()->defaultOrder()->orderBy('position')->get()->filter(function ($page) {
```

Comment:
Suggestion: Instead of `is_null($exceptPage) ? [] : [$exceptPage]`, you can use `array_filter([$exceptPage])` to remove all falsey values from an array.